README.md

```diff
@@ -24,6 +34,12 @@ This is the source for the Introduction to Data Science textbook.
 
 > ### Usage without RStudio
 > You can use this docker container to edit the files without RStudio using either vim or emacs. This can be done via:
+>
+> ```
+> docker-compose run --rm rstudio bash
```

Comment:
this shouldn't be in the "using without Rstudio" section, right?

Reply:
It should be, rstudio is just the nickname for the container. But now this makes me think this is not the best nickname... I'll change this to  `book-env`.

Reply:
90bf19c and 4184d39 address this.